application-accelerator/creating-accelerators/creating-accelerators.md

```diff
@@ -80,6 +80,127 @@ An alternative to using the Tanzu CLI is to create a separate manifest file and
+
+The accelerator samples include three fragments - `java-version`, `tap-initialize`, and `live-update`. See the [sample-accelerators/fragments](https://github.com/sample-accelerators/fragments/tree/tap-1.2) Git repository for the content of these fragments.
+
+To discover what fragments are available to use, you can run the following command:
+
+```
+tanzu accelerator fragment list
+```
+
+We'll look a the `java-version` as an example. It contains the following `accelerator.yaml` file:
+
+```
+accelerator:
+  options:
+  - name: javaVersion
+    inputType: select
+    label: Java version to use
+    choices:
+    - value: "1.8"
+      text: Java 8
+    - value: "11"
+      text: Java 11
+    - value: "17"
+      text: Java 17
+    defaultValue: "11"
+    required: true
+
+engine:
+  merge:
+    - include: [ "pom.xml" ]
+      chain:
+      - type: ReplaceText
+        regex:
+          pattern: "<java.version>.*<"
+          with: "'<java.version>' + #javaVersion + '<'"
+    - include: [ "build.gradle" ]
+      chain:
+      - type: ReplaceText
+        regex:
+          pattern: "sourceCompatibility = .*"
+          with: "'sourceCompatibility = ''' + #javaVersion + ''''"
+    - include: [ "config/workload.yaml" ]
+      chain:
+      - type: ReplaceText
+        condition: "#javaVersion == '17'"
+        substitutions:
+          - text: "spec:"
+            with: "'spec:\n  build:\n    env:\n    - name: BP_JVM_VERSION\n      value: \"17\"'"
+```
+
+This fragment will contribute the following to any accelerator that imports it:
+
+1. An option named `javaVersion` with three choices `Java 8`, `Java 11`, and `Java 17`
+1. Three `ReplaceText` transforms:
+    - if the accelerator has a `pom.xml` file then what is specified for `<java.version>` will be replaced with the chosen version.
+    - if the accelerator has a `build.gradle` file then what is specified for `sourceCompatibility` will be replaced with the chosen version.
+    - if the accelerator has a `config/workload.yaml` file and the user selected "Java 17" then a build env entry of BP_JVM_VERSION will be inserted into the `spec:` section.
+
+To deploy new fragments to the accelerator system you can use the new `tanzu accelerator fragment create` CLI command or you can apply a custom resource manifest file with either `kubectl apply` or the `tanzu accelerator apply` commands.
+
+The resource manifest for the `java-version` fragment looks like this:
+
+```
+apiVersion: accelerator.apps.tanzu.vmware.com/v1alpha1
+kind: Fragment
+metadata:
+  name: java-version
+  namespace: accelerator-system
+spec:
+  displayName: Select Java Version
+  git:
+    ref:
+      branch: tap-1.2
+    url: https://github.com/sample-accelerators/fragments.git
+    subPath: java-version
+```
+
+To create the fragment (we can save the above manifest in a `java-version.yaml` file) and use:
+
+```
+tanzu accelerator apply -f ./java-version.yaml
```

Comment:
You use `tanzu acc apply` to create fragments?  This initially confused me, so if this is true, you might want to clearly call out that the above command can be used for fragments and accelerators.

Reply:
I pushed another commit with a note to clarify